README.md

```diff
@@ -18,6 +18,13 @@ Currently supported cloud middleware:
 * OpenStack
 * OpenStack/ooi
 
+## Acknowledgement
+
+This work is co-funded by the [EOSC-hub project](http://eosc-hub.eu/) (Horizon 2020) under Grant number 777536.
+<img src="https://wiki.eosc-hub.eu/download/attachments/1867786/eu%20logo.jpeg?version=1&modificationDate=1459256840098&api=v2" height="24">
```

Comment:
Please use Markdown tags not raw HTML

Reply:
but then I cannot specify height...

Reply:
Arf, I really don't like this much it looks quite ugly.
But as far as I can see there aren't more alternatives apart maybe embedding/hosting an image at the right side.
So ok let's merge it to not loose to much time with this.